zerver/lib/home.py

```diff
@@ -158,6 +158,9 @@ def build_page_params_for_home_page_load(
         # The spectator client will be fetching the /register response
         # for spectators via the API. But we still need to set the
         # values not presence in that object.
+        #
+        # TODO: Raise `JsonableError` once we start using `do_events_register`
+        # for spectators.
```

Comment:
I think we already migrated spectators to use `do_events_register` via the API; so this comment is probably stale.